app/Repositories/Doctrine/AccountRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Repositories\Doctrine;
+
+use App\Database\Entities\Account;
+use App\Database\Exceptions\InvalidSubscriptionTypeException;
+use App\Repositories\Doctrine\ORM\AbstractRepository;
+use App\Repositories\Interfaces\AccountRepositoryInterface;
+
+final class AccountRepository extends AbstractRepository implements AccountRepositoryInterface
+{
+    /**
+     * Find by subscription type.
+     *
+     * @param string $subscriptionType
+     *
+     * @return mixed[]
+     */
+    public function findBySubscriptionType(string $subscriptionType): array
+    {
+        try {
+            if (($subscriptionType !== Account::SUBSCRIPTION_TYPE_MONTHLY) and
```

Comment:
We dont use 'and' as a logical operator, and we prefer '&&' instead, because it has a higher precedence compared to 'and'.